ugvc/mrd/ppmSeq_utils.py

```diff
@@ -773,17 +775,19 @@ def read_trimmer_tags_dataframe(
         df_tags = df_strand_ratio_category.drop(ppmSeqCategories.END_UNREACHED.value, errors="ignore")[
             [HistogramColumnNames.STRAND_RATIO_CATEGORY_START.value]
         ]
-        df_tags.index = [f"% {x} reads" for x in df_tags.index]
+        df_tags.index = [f"PCT_{x}_reads" for x in df_tags.index]
```

Comment:
better pct_ then PCT_ but it doesn't really matter

Reply:
I chose to be consistent with the Sorter metrics because I present them in the same table as the ppmSeq stats